data/campaigns/Son_Of_The_Black_Eye/scenarios/12_Giving_Some_Back.cfg

```diff
@@ -123,26 +123,26 @@
         [/part]
 
         [part]
-            story= _ "After a fair amount of squabbling — for some of the older warlords were reluctant to let this young upstart lead them — and a few consequent executions, the leadership of the horde was formally bestowed upon Kapou’e."
+            story= _ "After a fair amount of squabbling — for some of the older warlords were reluctant to let this young upstart lead them — and a few consequent executions, leadership of the horde was formally bestowed upon Kapou’e."
             {SOTBE_BIGMAP}
             {JOURNEY_11_OLD}
         [/part]
 
         [part]
-            story= _ "Riders were dispatched in all directions to gather intelligence and to pinpoint the exact location of the hostile forces."
+            story= _ "Riders were dispatched in all directions to gather intelligence and to pinpoint the exact location of hostile forces."
             {SOTBE_BIGMAP}
             {JOURNEY_11_OLD}
         [/part]
 
         [part]
-            story= _ "The horde was then split into two forces. On the advice of the shamans, Kapou’e placed one force under the leadership of Shan Taum the Smug, who — despite his obnoxious nature — was a fierce and capable leader. He was sent to Bitok, the most southwestern orcish city, to cut off the humans already invading the harbor of Tirigaz, and to stop any further human incursion into orcish territory."
+            story= _ "The horde was then split into two forces. On the advice of the shamans, Kapou’e placed one force under the leadership of Shan Taum the Smug, who — despite his obnoxious nature — was a fierce and capable leader. He was sent to Bitok, the most southwestern orcish city, to cut off the humans already invading the Harbor of Tirigaz, and to stop any further human incursion into orcish territory."
```

Comment:
Same as for S5, harbour should be left lower-case (95a5bba9889ebe82ee412d096342879684280026), or maybe say 'port city of Tirigaz' instead.

Reply:
Agreed.